docs/pages/docs/reference/command-line-reference.mdx

```diff
@@ -272,7 +272,7 @@ turbo run build --output-logs=new-only
 
 #### `--only`
 
-Default `false`. Restricts execution to only include specified tasks. This is very similar to how how `lerna` or `pnpm` run tasks by default.
+Default `false`. Restricts execution to only include specified tasks. This is very similar to how `lerna` and `pnpm` run tasks by default.
```

Comment:
```suggestion
Default `false`. Restricts execution only to include specified tasks. This is very similar to how `lerna` and `pnpm` run tasks by default.
```

Reply:
🤔 . Neither sounds very good to me. how about "Restrictions execution to include specific tasks only" ?